### PR TITLE
Network perf v2

### PR DESCRIPTION
**Pull request by paigerube14**
Adding benchmark comparison call 

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/network-perf-v2/24/console

### CODE REVIEW

**Comment by qiliRedHat** [comment]
@SachinNinganure Would PTAL?

File: Jenkinsfile [hunk 1]
```diff
@@ -58,6 +58,29 @@ pipeline {
       name: 'WORKLOAD_TYPE', 
       description: 'Workload type'
     )
+    string(
+        name: "COMPARISON_CONFIG",
+        defaultValue: "k8s-touchstone.json",
```
Comment (qiliRedHat) [inline Jenkinsfile:63]: @paigerube14 I did not see k8s-touchstone.json under https://github.com/paigerube14/benchmark-comparison/tree/master/config. Same for k8s-tolerancy.yaml below. Where are they from?

**Comment by qiliRedHat** [comment]
I saw some ERROR in https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/benchmark-comparison/573/console
```
10-31 03:12:29.927  [1mMon Oct 30 19:12:29 UTC 2023 Running: touchstone_compare --database elasticsearch -url https://ocp-qe:****@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com/ https://ocp-qe:****@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com/ -u False bc1143a2-64ce-429d-b7b7-0acbad47f59a --config /tmp/network-perf-v2-bc1143a2-64ce-429d-b7b7-0acbad47f59a/k8s-touchstone.json --tolerancy-rules /home/jenkins/ws/workspace/ch-pipeline_benchmark-comparison/tolerancy-configs/k8s-tolerancy.yaml -o csv --output-file /home/jenkins/ws/workspace/ch-pipeline_benchmark-comparison/e2e-benchmarking/utils/network-perf-v2-bc1143a2-64ce-429d-b7b7-0acbad47f59a.csv[0m
10-31 03:12:30.922  2023-10-30 19:12:30,786 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_STREAM'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
10-31 03:12:31.846  2023-10-30 19:12:31,613 - touchstone - ERROR - Missing UUID in input dict: {'bc1143a2-64ce-429d-b7b7-0acbad47f59a': 4925.18994140625}
10-31 03:12:31.846  2023-10-30 19:12:31,613 - touchstone - ERROR - Missing UUID in input dict: {'bc1143a2-64ce-429d-b7b7-0acbad47f59a': 4874.39990234375}
10-31 03:12:31.846  2023-10-30 19:12:31,673 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'UDP_STREAM'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
10-31 03:12:31.846  2023-10-30 19:12:31,731 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'UDP_STREAM'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
10-31 03:12:31.846  2023-10-30 19:12:31,797 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_CRR'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
10-31 03:12:32.101  2023-10-30 19:12:31,849 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_CRR'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
10-31 03:12:32.101  2023-10-30 19:12:31,904 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_RR'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
10-31 03:12:32.101  2023-10-30 19:12:31,958 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_RR'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
10-31 03:12:32.101  [1mMon Oct 30 19:12:32 UTC 2023 compare result: 0[0m
```

File: Jenkinsfile (hunk 1 above)
Comment (paigerube14) [reply Jenkinsfile:63]: the files used in this comparison is from [here](https://github.com/openshift-qe/ocp-qe-perfscale-ci/tree/benchmark-comparison/configs)

**Comment by paigerube14** [comment]
> I saw some ERROR in https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/benchmark-comparison/573/console
> 
> ```
> 10-31 03:12:29.927  �[1mMon Oct 30 19:12:29 UTC 2023 Running: touchstone_compare --database elasticsearch -url https://ocp-qe:****@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com/ https://ocp-qe:****@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com/ -u False bc1143a2-64ce-429d-b7b7-0acbad47f59a --config /tmp/network-perf-v2-bc1143a2-64ce-429d-b7b7-0acbad47f59a/k8s-touchstone.json --tolerancy-rules /home/jenkins/ws/workspace/ch-pipeline_benchmark-comparison/tolerancy-configs/k8s-tolerancy.yaml -o csv --output-file /home/jenkins/ws/workspace/ch-pipeline_benchmark-comparison/e2e-benchmarking/utils/network-perf-v2-bc1143a2-64ce-429d-b7b7-0acbad47f59a.csv�[0m
> 10-31 03:12:30.922  2023-10-30 19:12:30,786 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_STREAM'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
> 10-31 03:12:31.846  2023-10-30 19:12:31,613 - touchstone - ERROR - Missing UUID in input dict: {'bc1143a2-64ce-429d-b7b7-0acbad47f59a': 4925.18994140625}
> 10-31 03:12:31.846  2023-10-30 19:12:31,613 - touchstone - ERROR - Missing UUID in input dict: {'bc1143a2-64ce-429d-b7b7-0acbad47f59a': 4874.39990234375}
> 10-31 03:12:31.846  2023-10-30 19:12:31,673 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'UDP_STREAM'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
> 10-31 03:12:31.846  2023-10-30 19:12:31,731 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'UDP_STREAM'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
> 10-31 03:12:31.846  2023-10-30 19:12:31,797 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_CRR'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
> 10-31 03:12:32.101  2023-10-30 19:12:31,849 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_CRR'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
> 10-31 03:12:32.101  2023-10-30 19:12:31,904 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_RR'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
> 10-31 03:12:32.101  2023-10-30 19:12:31,958 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'profile': 'TCP_RR'}, 'buckets': ['hostNetwork', 'messageSize', 'samples', 'parallelism', 'acrossAZ', 'service'], 'aggregations': {'throughput': ['avg']}}
> 10-31 03:12:32.101  �[1mMon Oct 30 19:12:32 UTC 2023 compare result: 0�[0m
> ```

Any errors seen in benchmark-comparison will need to be updated on the benchmark-comparison side, I think that issue only happened because it didnt find a baseline uuid to compare to.
This pr is only adding the ability to call the comparison job

**Pull request merged**